mender-convert-modify

```diff
@@ -164,6 +164,16 @@ if [ "${MENDER_GRUB_EFI_INTEGRATION}" == "y" ]; then
 
 fi
 
+run_and_log_cmd "sudo git init work/meta-mender"
+run_and_log_cmd "sudo git -C work/meta-mender remote add origin https://github.com/mendersoftware/meta-mender.git"
+run_and_log_cmd "sudo git -C work/meta-mender fetch --depth 1 origin e05cde10fde3646a78081709e717fb42a6c5ba44"
```

Comment:
Q:
Why not get this file with `curl` against github? :thinking:

Reply:
?

Reply:
Ah, nvm

Reply:
Could curl it from https://raw.githubusercontent.com/mendersoftware/meta-mender/master/meta-mender-core/recipes-mender/mender-client/files/mender-client-data-dir.service
But no guarantee that url won't change in the future

Reply:
Yes, I agree :)